cleanlab/multiannotator.py

```diff
@@ -126,6 +139,13 @@ def get_label_quality_multiannotator(
             * ``num_examples_labeled``: number of examples annotated by a given annotator.
             * ``agreement_with_consensus``: fraction of examples where a given annotator agrees with the consensus label.
             * ``worst_class``: the class that is most frequently mislabeled by a given annotator.
+
+        ``model_weight`` : float (returned if `return_weights=True`)
+            float specifying the model weight used in weighted averages (only applicable for ``quality_method == crowdlab``), ``None`` otherwise
+
+        ``annotator_weight`` : np.ndarray (returned if `return_weights=True`)
+            An array of shape ``(M,)`` where M is the number of annotators, specifying the annotator weights used in weighted averages
```

Comment:
```suggestion
        ``model_weight`` : float
            Only returned if `return_weights=True`, this number specifies the weight of classifier model in weighted averages used to estimate label quality (only applicable for ``quality_method == crowdlab``), ``None`` otherwise. This number is an estimate of how trustworthy the model is relative the annotators.

        ``annotator_weight`` : np.ndarray (returned if `return_weights=True`)
            An array of shape ``(M,)`` where M is the number of annotators, specifying the weight of each annotator in weighted averages used to estimate label quality.  These weights are estimates of how trustworthy each annotator is relative to the other annotators.
```